packages/server/src/errors.ts

```diff
@@ -1,179 +1,40 @@
-  }
-
-  override toString(): string {
-    return printError(toGraphQLError(this));
-  }
-
-  get [Symbol.toStringTag](): string {
-    return this.name;
-  }
-}
-
-function toGraphQLError(error: ApolloError): GraphQLError {
-  return new GraphQLError(
-    error.message,
-    error.nodes,
-    error.source,
-    error.positions,
-    error.path,
-    error.originalError,
-    error.extensions,
-  );
-}
-
-function enrichError(error: Partial<GraphQLError>, debug: boolean = false) {
-  // follows similar structure to https://github.com/graphql/graphql-js/blob/main/src/error/GraphQLError.ts#L127-L176
-  // with the addition of name
-  const expanded = Object.create(Object.getPrototypeOf(error), {
-    name: {
-      value: error.name,
-    },
-    message: {
-      value: error.message,
-      enumerable: true,
-      writable: true,
-    },
-    locations: {
-      value: error.locations || undefined,
-      enumerable: true,
-    },
-    path: {
-      value: error.path || undefined,
-      enumerable: true,
-    },
-    nodes: {
-      value: error.nodes || undefined,
-    },
-    source: {
-      value: error.source || undefined,
-    },
-    positions: {
-      value: error.positions || undefined,
-    },
-    originalError: {
-      value: error.originalError,
-    },
-  });
-
-  expanded.extensions = {
-    ...error.extensions,
-    code: error.extensions?.code || 'INTERNAL_SERVER_ERROR',
-    exception: {
-      ...error.extensions?.exception,
-      ...(error.originalError as any),
-    },
-  };
-
-  // ensure that extensions is not taken from the originalError
-  // graphql-js ensures that the originalError's extensions are hoisted
-  // https://github.com/graphql/graphql-js/blob/0bb47b2/src/error/GraphQLError.js#L138
-  delete expanded.extensions.exception.extensions;
-  if (debug && !expanded.extensions.exception.stacktrace) {
-    const stack = error.originalError?.stack || error.stack;
-    expanded.extensions.exception.stacktrace = stack?.split('\n');
-  }
-
-  if (Object.keys(expanded.extensions.exception).length === 0) {
-    // remove from printing an empty object
-    delete expanded.extensions.exception;
-  }
-
-  return expanded as ApolloError;
-}
-
-export function toApolloError(
-  error: Error & { extensions?: Record<string, any> },
-  code: string = 'INTERNAL_SERVER_ERROR',
-): Error & { extensions: Record<string, any> } {
-  let err = error;
-  if (err.extensions) {
-    err.extensions.code = code;
-  } else {
-    err.extensions = { code };
-  }
-  return err as Error & { extensions: Record<string, any> };
-}
-
 export interface ErrorOptions {
   code?: string;
   // This declaration means it takes any "class" that has a constructor that
   // takes a single string, and should be invoked via the `new` operator.
-  errorClass?: new (message: string) => ApolloError;
+  errorClass?: new (message: string) => GraphQLError;
 }
 
 export function fromGraphQLError(error: GraphQLError, options?: ErrorOptions) {
```

Comment:
We're down to just one call to fromGraphQLError. I believe you said you are planning to get rid of it, right?